index.bs

```diff
@@ -2632,8 +2632,7 @@ After the Review Period</h4>
 	whether there were any [=Formal Objections=],
 	with attention to <a href="#confidentiality-change">changing the confidentiality level</a> of the [=Formal Objections=].
 
-	If there is [=dissent=]
-	(i.e., there were [=Formal Objections=], at least some of which were sustained)
+	If there were [=Formal Objections=], at least some of which were sustained,
```

Comment:
There was discussion earlier, that dissent could exist without Formal Objections, which seems to be overwritten by the PR, as written. This tweak doesn't quite put less-intense dissent back, but it at least doesn't say that sustained Formal Objections are the *only* possible form of dissent....
```suggestion
	If there is [=dissent=]
	(e.g., there were [=Formal Objections=], at least some of which were sustained),
```

Reply:
This tweak says that one dissenting member can force the proposal to be rejected or returned for more work, even if their formal objection is overruled. I don't think that's what we want the Process to say?

Reply:
@jyasskin — I don't understand how you get that meaning.

With `i.e.`, it's "If there is dissent (that is, there were Formal Objections, at least some of which were sustained)".  In other words, dissent can *only* take the form of [sustained] Formal Objections.

With `e.g.`, it's "If there is dissent (for example, there were Formal Objections, at least some of which were sustained)".  In other words, dissent *may* take the form of [sustained] Formal Objections, but there are also other forms of dissent which may apply in this section.

(Here we have another reason to avoid `i.e.` and `e.g.,` even though the previous advice from the i18n folks has been reconsidered.)

Reply:
Right, so your use of `e.g.` is an improvement with respect to the meaning of "dissent", but I don't think it results in the right meaning of the whole paragraph.

The [whole paragraph](https://www.w3.org/Consortium/Process/Drafts/#ACReviewAfter) we're discussing is:

> If there is [dissent](https://www.w3.org/Consortium/Process/Drafts/#def-Dissent) (...) or ..., [W3C Decision](https://www.w3.org/Consortium/Process/Drafts/#def-w3c-decision) must be one of:
> * The proposal is returned for additional work, with a request to the initiator to improve the proposal.
> * The proposal is rejected.

Dissent is created by a single member. There's a statement in the [previous section](https://www.w3.org/Consortium/Process/Drafts/#ACReviewStart) that dissent in this context must be expressed as a Formal Objection, but whether or not we agree with that, the existence of the dissent doesn't depend on whether the FO Council agrees with that member. But the paragraph then says that this single member can force the W3C Decision to be to either reject or return the proposal. That doesn't seem right to me.

The current wording uses `i.e.` to (badly) redefine "dissent" so that the rest of the paragraph works: it says that if some Formal Objections were sustained, then the proposal has to be returned or rejected. Changing the `i.e.` to `e.g.` avoids redefining "dissent" (good), but then gives single dissenters too much power (bad). My suggestion is to remove the use of "dissent" in this paragraph, since the W3C Decision depends on the disposition of the FO, not on whether someone disagrees.

Reply:
> My suggestion is to remove the use of "dissent" in this paragraph, since the W3C Decision depends on the disposition of the FO, not on whether someone disagrees.

I think that's a good direction.